### ISSUE TITLE
Accessibility fixes for 'contact us' form

### ISSUE THREAD

**Issue by mel-miller** [badcamp/badcamp-2019#65]
Issue derived from: Accessibility Audit (#27)

Contact Form: https://2019.badcamp.org/contact-us

Input field has no description: WCAG 1.3.1 Info and Relationships

How to fix it:
If the input field has a visible description indicating the purpose of the field, this description should be explicitly associated to the input field either as a HTML label or using the WAI-ARIA attribute 'aria-labelledby'.
If it is not possible to add a visible description, either add a mouseover text ('title' attribute) to the input field or create an invisible label using the WAI-ARIA attribute 'aria-label'.

**Comment by AmyJuneH** [comment]
https://www.drupal.org/project/badcamp/issues/3058554
